src/appleseed/renderer/kernel/lighting/directlightingintegrator.cpp

```diff
@@ -180,17 +178,14 @@ void DirectLightingIntegrator::compute_outgoing_radiance_light_sampling(
     }
 
     if (m_light_sample_count > 1)
-    {
-        const float rcp_light_sample_count = 1.0f / m_light_sample_count;
-        radiance *= rcp_light_sample_count;
-    }
+        radiance /= m_light_sample_count;
```

Comment:
You need to static_cast `m_light_sample_count` to `float`, otherwise VC++ will emit a warning (rightfully! gcc should emit one too!).